services/web/server/src/simcore_service_webserver/projects/projects_service.py

```diff
@@ -988,17 +1001,17 @@ async def is_project_hidden(app: web.Application, project_id: ProjectID) -> bool
     return await db.is_hidden(project_id)
 
 
-async def patch_project_node(
+async def update_project_node(
     app: web.Application,
     *,
     product_name: ProductName,
     user_id: UserID,
     project_id: ProjectID,
     node_id: NodeID,
-    node_patch: NodePatch,
+    partial_node: PartialNode,
 ) -> None:
     _node_patch_exclude_unset: dict[str, Any] = jsonable_encoder(
-        node_patch, exclude_unset=True, by_alias=True
+        partial_node, exclude_unset=True, by_alias=True
     )
```

Comment:
the naming here is confusing.
patch you changed, but then it's again a patch and the function is called update. Please choose one and stick to it?

Reply:
cant you do this with just 
```python
_node_patch_exclude_unset = PartialNode.model_dump(mode='json', exclude_unset=True, by_alias=True)
```